Bridgecraft/main.swift

```diff
@@ -88,12 +88,12 @@ func bridgingHeaderPath(projectURL: URL, targetName: String) throws -> String {
         }
     }
     
-    guard headerPath != nil else {
+    guard let headerPathUnwrapped = headerPath else {
```

Comment:
I think it's a good practice to call the unwrapped variable the same as the optional, could you change it?

Reply:
It's not possible to do that with local variables, only with global variables or function arguments.

Here is the error when trying to do that:
![image](https://user-images.githubusercontent.com/784666/35193583-a15f20f0-fe9c-11e7-8fbc-aa1ba112767e.png)

I can use a different naming pattern if you prefer, but the name needs to be different.
https://stackoverflow.com/a/41854563

Reply:
true, I missed that